### PR TITLE
2 packages from sapristi/easy_logging at 0.6.2

### PR DESCRIPTION
**Pull request by sapristi**
This pull-request concerns:
-`easy_logging.0.6.2`: Module to log messages. Aimed at being both powerful and easy to use
-`easy_logging_yojson.0.6.2`: Configuration loader for easy_logging with yojson backend

---
* Homepage: https://sapristi.github.io/easy_logging/
* Source repo: git+https://github.com/sapristi/easy_logging.git
* Bug tracker: https://github.com/sapristi/easy_logging/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by sapristi** [comment]
It seems like the download of the calendar package sources failed.

File: packages/easy_logging_yojson/easy_logging_yojson.0.6.2/opam [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+license: "GPL"
+homepage: "https://sapristi.github.io/easy_logging/"
+bug-reports: "https://github.com/sapristi/easy_logging/issues"
+dev-repo: "git+https://github.com/sapristi/easy_logging.git"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {>= "1.8"}
+  "ppx_deriving"{>= "4.0" & < "5.0"}
+  "ppx_deriving_yojson"
+  "easy_logging" {= "0.6.2"}
```
Comment (kit-ty-kate) [inline packages/easy_logging_yojson/easy_logging_yojson.0.6.2/opam]:
```suggestion
  "easy_logging" {= version}
```

Comment (kit-ty-kate) [reply packages/easy_logging_yojson/easy_logging_yojson.0.6.2/opam]: this is a suggestion only, I'll merge it if it passes the CI before

**Comment by sapristi** [comment]
Thanks for the suggestion @kit-ty-kate, I applied the diff to the PR.

**Comment by mseri** [comment]
Thanks

**Pull request merged**